SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -277,6 +329,13 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 			: SpatialOSLaunchConfig.FilePath;
 	}
 
+	FORCEINLINE FString GetSpatialOSConfig()
+	{
+		return SpatialOSLaunchConfig.FilePath.IsEmpty()
+			? FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectDir(), TEXT("/../spatial/spatialos.json")))
+			: SpatialOSLaunchConfig.FilePath;
```

Comment:
Why is this returning the launch config if the spatialos.json doesn't exist?

Reply:
Good catch - removed the method anyway 🥉.